public/app/features/plugins/admin/components/PluginListRow.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { Icon, useStyles2, CardContainer, HorizontalGroup, VerticalGroup, Tooltip } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { CatalogPlugin, IconName } from '../types';
+import { PluginLogo } from './PluginLogo';
+import { PluginListBadges } from './PluginListBadges';
+
+const LOGO_SIZE = '48px';
+
+type PluginListRowProps = {
+  plugin: CatalogPlugin;
+  pathName: string;
+};
+
+export function PluginListRow({ plugin, pathName }: PluginListRowProps) {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <CardContainer href={`${pathName}/${plugin.id}`} className={styles.cardContainer}>
+      <VerticalGroup spacing="md">
+        <div className={styles.headerWrap}>
+          <PluginLogo
+            src={plugin.info.logos.small}
+            alt={`${plugin.name} logo`}
+            className={styles.image}
+            height={LOGO_SIZE}
+          />
+          <div>
+            <h3 className={styles.name}>{plugin.name}</h3>
+            <p className={styles.orgName}>By {plugin.orgName}</p>
+            <HorizontalGroup height="auto">
+              <PluginListBadges plugin={plugin} />
+              {plugin.hasUpdate && !plugin.isCore ? (
+                <Tooltip content={plugin.version}>
+                  <p className={styles.hasUpdate}>Update available!</p>
+                </Tooltip>
+              ) : null}
```

Comment:
**Question:** could we use a simple `&&` instead of the ternary?

```javascript
{plugin.hasUpdate && !plugin.isCore && (
    <Tooltip content={plugin.version}>
        <p className={styles.hasUpdate}>Update available!</p>
    </Tooltip>
)}
```

Reply:
Changed it in https://github.com/grafana/grafana/pull/39466/commits/248e3ba851ea95542fb2540be395cad94612fb90.